src/Workspaces/Core/Portable/Workspace/IWorkspaceAccessorService.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis;
+
+internal interface IWorkspaceAccessorService : IWorkspaceService
+{
+    Workspace Workspace { get; }
```

Comment:
this seems really strange to have a service to get this.  it feels like we're not passing through data we should be if we need this.

Reply:
Accessing through `HostWorkspaceServices` was also available, but marked obsolete. This interface allows us to run a quick speedometer validation without spending too much time refactoring this.